packages/nimble-components/src/wafer-map/tests/DataGenerator.ts

```diff
@@ -0,0 +1,56 @@
+import type { WaferMapDie } from '../types';
+import type { IValueGenerator } from './ValueGenerator';
+
+const valueToString = (value: number): string => {
+    return value % 1 ? value.toFixed(2) : Math.trunc(value).toString();
+};
+
+export const generateWaferData = (
+    numDies: number,
+    valueGenerator?: IValueGenerator
+): WaferMapDie[] => {
+    const diesSet: WaferMapDie[] = [];
```

Comment:
we should create the seeded random number generator with a fixed seed each time this is called for reproducible data sets

Reply:
applied